src/GraphQL.NewtonsoftJson/GraphQL.NewtonsoftJson.csproj

```diff
@@ -0,0 +1,16 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netstandard2.0</TargetFramework>
+    <RootNamespace>GraphQL.NewtonsoftJson</RootNamespace>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Newtonsoft.Json" Version="12.0.2" />
```

Comment:
Maybe update to 12.0.3 ?

Reply:
@joemcbride I also remembered that I wanted to ask you what needs to be done so that a new project begins to fall into [artifacts](https://ci.appveyor.com/project/graphql-dotnet-ci/graphql-dotnet/builds/29906112/artifacts) and will be published to myget. I see that there are some JS scripts, but I understand almost nothing in them.

Reply:
Should be upgraded